com.mandelbulber.Mandelbulber.yaml

```diff
@@ -0,0 +1,115 @@
+app-id: com.mandelbulber.Mandelbulber
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: mandelbulber2
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  #- /include
+  #- /lib/cmake
+  #- /share/doc
+  #- /share/applications
+  #- *.la
+  #- *.a
+
+modules:
+  - name: libpng12
```

Comment:
libpng is part of Sdk, this is not needed unless Mandelbulber specifically requires an older libpng